crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -710,6 +712,17 @@ impl<F: Send + Clone> GetTracker<F, PaymentData<F>, api::PaymentsRequest> for Pa
                 payment_method_data_billing.get_billing_address()
             });
 
+        // validate billing name for card holder name
+        helpers::validate_billing_name(
```

Comment:
create a domain type which will have these validations, use the domain type wherever validations are required